shared/chat/inbox/row/simple-row.js

```diff
@@ -1,393 +1,108 @@
 // @flow
-import React, {PureComponent} from 'react'
-import {
-  Avatar,
-  Text,
-  MultiAvatar,
-  Icon,
-  Usernames,
-  Markdown,
-  Box,
-  ClickableBox,
-} from '../../../common-adapters'
-import {
-  globalStyles,
-  globalColors,
-  globalMargins,
-  styleSheetCreate,
-  collapseStyles,
-  lineHeight,
-} from '../../../styles'
-import {isMobile} from '../../../constants/platform'
-import memoize from 'lodash/memoize'
+import React from 'react'
+import {Box, ClickableBox} from '../../../common-adapters'
+import {globalStyles} from '../../../styles'
 import {List} from 'immutable'
 
-import type {ConversationIDKey} from '../../../constants/chat'
+import {SimpleTopLine, FilteredTopLine} from './top-line'
+import BottomLine from './bottom-line'
+import {Avatars} from './avatars'
 
-// All this complexity isn't great but the current implementation of avatar forces us to juggle all these colors and
-// forces us to explicitly choose undefined/the background/ etc. This can be cleaned up when avatar is simplified
-function rowBorderColor(idx: number, isLastParticipant: boolean, backgroundColor: string) {
-  // Only color the foreground items
-  if (isLastParticipant) {
-    return undefined
-  }
-
-  // We don't want a border if we're a single avatar
-  return !idx && isLastParticipant ? undefined : backgroundColor
-}
-
-type TopLineProps = {
+type SimpleProps = {
+  backgroundColor: string,
   hasUnread: boolean,
-  participants: List<string>,
-  teamname: ?string,
-  showBold: boolean,
-  subColor: ?string,
-  timestamp: ?string,
-  usernameColor: ?string,
-}
-
-class TopLine extends PureComponent<void, TopLineProps, void> {
-  render() {
-    const {hasUnread, showBold, participants, subColor, timestamp, usernameColor, teamname} = this.props
-    const height = isMobile ? 19 : 17
-    const boldOverride = showBold ? globalStyles.fontBold : null
-    return (
-      <Box style={{...globalStyles.flexBoxRow, alignItems: 'center', maxHeight: height, minHeight: height}}>
-        <Box
-          style={{
-            ...globalStyles.flexBoxRow,
-            flex: 1,
-            maxHeight: height,
-            minHeight: height,
-            position: 'relative',
-          }}
-        >
-          <Box
-            style={{
-              ...globalStyles.flexBoxColumn,
-              bottom: 0,
-              justifyContent: 'flex-start',
-              left: 0,
-              position: 'absolute',
-              right: 0,
-              top: 0,
-            }}
-          >
-            <Usernames
-              inline={true}
-              plainText={true}
-              type="BodySemibold"
-              plainDivider={isMobile ? undefined : ',\u200a'}
-              containerStyle={{...boldOverride, color: usernameColor, paddingRight: 7}}
-              users={teamname ? [{username: teamname}] : participants.map(p => ({username: p})).toArray()}
```

Comment:
we don't need this teamname logic anymore, right, since we branched those off into their own components?